src/SwedishBankAccounts.UnitTests/BankAccountNumberTests.cs

```diff
@@ -47,12 +45,23 @@ public class BankAccountNumberTests : UnitTests
     [InlineData("9892 398 7468", "Riksgälden")]
     [InlineData("9578 633 1128", "Sparbanken Syd")]
     [InlineData("9340 321 4681", "Swedbank")]
-    public void AccountNumber_TryParse_ShouldValidate(string accountNumber, string bankName)
+	public void BankAccountNumber_TryParse_ShouldValidate(string accountNumber, string bankName)
     {
         BankAccountNumber.TryParse(accountNumber, out var bankAccountNumber)
             .Should().BeTrue($"it should be validated to {bankName}");
 
         bankAccountNumber.Should().NotBeNull($"it should be validated to {bankName}");
         bankAccountNumber!.Bank.Should().Be(bankName, $"it should be validated to {bankName}");
     }
+
+    [Theory]
+    [InlineData("1234567890123456", InitOptions.Strict)]
```

Comment:
Add an InlineData with same accountNumber as Lax in previous test but here as Strict that should fail